frontend/src/components/common/chip/chip.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react';
+import { ChipStyle } from 'common/enums/enums';
+import { getValidClasses } from 'helpers/helpers';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  value: string;
+  chipStyle?: ChipStyle;
+};
+
+const Chip: FC<Props> = ({ value, chipStyle = ChipStyle.FILLED }) => {
+  const fullClassName = getValidClasses(
+    styles.chip,
+    styles[`chipStyle${chipStyle}`],
+  );
+
+  return <button className={fullClassName}>{value}</button>;
```

Comment:
```suggestion
  return <span className={fullClassName}>{value}</span>;
```
let's leave as span for now